packages/cinterop/src/jvmCommon/kotlin/io/realm/interop/gc/NativeContext.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2021 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.interop.gc
+
+import io.realm.interop.LongPointerWrapper
+import java.lang.ref.ReferenceQueue
+
+object NativeContext {
```

Comment:
This is slightly different from Java where some pointers like `SharedRealm` are not _dtor_ Thread safe. Since we're not using `SharedRealm` & `Table` API in the C-API I don't see why we should reintroduce the dummy context.